benchmarks/init_middleware.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+var http = require('http')
+var Benchmark = require('benchmark')
+var suite = new Benchmark.Suite()
+var IncomingMessage = http.IncomingMessage
+var ServerResponse = http.ServerResponse
```

Comment:
Since we are trying hard to move to a more agnostic http impl, can you change these references to the exports from express instead of the core exports?